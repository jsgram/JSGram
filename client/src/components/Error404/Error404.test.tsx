import React from 'react';
import { Error404 } from './index';
import { shallow } from 'enzyme';
import { BrowserRouter } from 'react-router-dom';

describe('Error404 component', () => {
    let renderer;

    beforeEach(() => {
        renderer = shallow(<BrowserRouter><Error404/>)</BrowserRouter>);
    });

    test('render-success', () => {
        expect(renderer.html()).toHaveLength(222);
    });
});
