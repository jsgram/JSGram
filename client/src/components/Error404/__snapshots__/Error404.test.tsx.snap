// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error404 component render-success 1`] = `ShallowWrapper {}`;
